ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -77,6 +77,36 @@
       "Minimum time between consecutive health check evaluations. Health check queries before "
           + "the interval has elapsed will receive cached responses.";
 
+  public static final String KSQL_HEARTBEAT_SEND_INTERVAL_MS_CONFIG =
+      KSQL_CONFIG_PREFIX + "heartbeat.send.interval.ms";
+  private static final String KSQL_HEARTBEAT_SEND_INTERVAL_MS_DOC =
+      "Interval at which heartbeats are broadcasted to servers.";
+
+  public static final String KSQL_HEARTBEAT_CHECK_INTERVAL_MS_CONFIG =
```

Comment:
check is a better name. also rename `ProcessHeartBeatTask` to `CheckHeartBeatTask`?